### PR TITLE
Create a models

### PR DESCRIPTION
**Pull request by YuraFesun**
Create a models: EventType, Event, Storage, ActionType

### CODE REVIEW

File: src/main/java/Model/Event.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package Model;
+
+import java.util.Date;
+
+public class Event {
+    private EventType eventType;
+    private String name;
+    private int ticketsQuantity;
+    private Date date;
```
Comment (AndrewPolishchuc) [inline src/main/java/Model/Event.java]: mb LocalDateTime?

File: src/main/java/Model/Storage.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package Model;
+
+import java.util.LinkedList;
+
+public class Storage {
+    private LinkedList<Event> eventList;
```
Comment (AndrewPolishchuc) [inline src/main/java/Model/Storage.java]: Why u use LinkedList?

**Review by AndrewPolishchuc — CHANGES_REQUESTED** [review]

File: src/main/java/Model/Event.java (hunk 1 above)
Comment (YuraFesun) [reply src/main/java/Model/Event.java]: Yes, sorry

File: src/main/java/Model/Storage.java [hunk 3]
```diff
@@ -1,9 +1,10 @@
 package Model;
 
+import java.util.HashMap;
 import java.util.LinkedList;
 
 public class Storage {
-    private LinkedList<Event> eventList;
+    private HashMap<String,Event> eventList;
```
Comment (AndrewPolishchuc) [inline src/main/java/Model/Storage.java]: private HashMap<String, Event> eventList;

**Review by AndrewPolishchuc — CHANGES_REQUESTED** [review]

File: src/main/java/Model/Storage.java (hunk 3 above)
Comment (AndrewPolishchuc) [reply src/main/java/Model/Storage.java]: how many events can there be with the same name and different dates?

Comment (AndrewPolishchuc) [reply src/main/java/Model/Storage.java]: You should use a different structure for map value

**Review by AndrewPolishchuc — APPROVED** [review]

**Pull request merged**